src/plugins/mer/Mer.json.in

```diff
@@ -1,5 +1,5 @@
 {
-    \"Name\" : \"SailfishOS\",
+    \"Name\" : \"Mer\",
```

Comment:
It is not desired anymore to present the plugin under its original name "Mer", so this cannot be done. It could be possible to rename the plugin to match the name declared here.